Tutorial.playground/Contents.swift

```diff
@@ -0,0 +1,662 @@
+// Copyright (c) 2012-2014 Gabriel Gonzalez
+// All rights reserved.
+//	
+// Redistribution and use in source and binary forms, with or without modification,
+// are permitted provided that the following conditions are met:
+// * Redistributions of source code must retain the above copyright notice,
+// this list of conditions and the following disclaimer.
+// * Redistributions in binary form must reproduce the above copyright notice,
+// this list of conditions and the following disclaimer in the documentation
+// and/or other materials provided with the distribution.
+// * Neither the name of Gabriel Gonzalez nor the names of other contributors
+// may be used to endorse or promote products derived from this software
+// without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+// ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+// WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+// DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+// ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+// (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+// LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+// ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+
+// MARK: - Introduction
+
+//: The `Aquifer` library decouples stream processing stages from each other so
+//: that you can mix and match diverse stages to produce useful streaming
+//: programs.  If you are a library writer, `Aquifer` lets you package up
+//: streaming components into a reusable interface.  If you are an application
+//: writer, `Aquifer` lets you connect pre-made streaming components with minimal
+//: effort to produce a highly-efficient program that streams data in constant
```

Comment:
I'm not sure of any of the claims in this sentence.  Aquifer will use "constant memory" in the sense that it will "constantly use all of the stack" in some cases.  And it's going to be relatively inefficient without compiler-aides fusion